rhombus-lib/rhombus/private/amalgam/annotation.rkt

```diff
@@ -442,11 +442,12 @@
    (lambda (form tail)
      (syntax-parse tail
        [(op::name . (~var t (:annotation-seq #'::)))
+        (define op-name (string->symbol (shrubbery-syntax->string #'op.name)))
```

Comment:
Is the purpose of this change to use the raw text as the `who`?

Reply:
Yes, that's right.